### ISSUE TITLE
Port request: OpenID

### ISSUE THREAD

**Issue by robbrandt** [backdrop-ops/contrib#829]
**Name of the module, theme, or layout**
OpenID

**Link to the drupal.org module, theme, or layout**
https://www.drupal.org/docs/7/core/modules/openid

**(Optional) Link to an issue in the drupal.org queue asking the Drupal community if anyone is working on a Backdrop port of this project**
None found

**(Optional) Bounty**
We are a poverty stricken non-profit lol

**Comment by klonos** [comment]
Is this about https://www.drupal.org/project/openid or https://www.drupal.org/project/openid_connect?

**Comment by argiepiano** [comment]
This is about the **core** OpenID module, that was removed from Backdrop.

**Comment by robbrandt** [comment]
> This is about the **core** OpenID module, that was removed from Backdrop.

Correct, https://www.drupal.org/project/openid .  We have an older private OpenID server that is still server our purposes well in our Drupal 7 implementation that we need to migrate to something else.

**Comment by argiepiano** [comment]
> Correct, https://www.drupal.org/project/openid . 

@robbrandt, the link you pasted is not the core OpenID module. That's and old contrib module with the same name, that runs on Drupal 5 (!). The correct link for the D7 module is what I pasted above.
